subscription-service/src/main/kotlin/com/egm/stellio/subscription/service/SubscriptionService.kt

```diff
@@ -76,6 +76,7 @@ class SubscriptionService(
             .bind("is_active", subscription.isActive)
             .bind("expires_at", subscription.expiresAt)
             .bind("sub", sub.toStringValue())
+            .bind("contexts", subscription.contexts.joinToString(separator = ","))
```

Comment:
why not storing the contexts as an array?

Reply:
I change it